WiseBuy/View Controllers/ComposeViewController.m

```diff
@@ -60,4 +74,22 @@ - (IBAction)didTapPost:(id)sender {
     }];
 }
 
+- (BOOL) inputFieldsAreEmpty {
+    return [self.itemName.text isEqual:@""]  || [self.price.text isEqual:@""]  || [self.sellerName.text isEqual:@""] || [self.buyLink.text isEqual:@""];
```

Comment:
Nit: newlines would make it easier to read

General rule of thumb is if you have a condition with multiple parts (separated by || or &&) and it's too long to fit on a single line (~80 chars), put each condition on its own line.